### ISSUE TITLE
Bad accuracy with the Adadelta optimizer for the mnist example

### ISSUE THREAD

**Issue by MattAbrs** [GreenWaves-Technologies/gap_sdk#217]
Hi,

I'm actually working with the 3.9_dev SDK version.

When I execute the train of the mnist example, I get a 0,10 accuracy.
I tried to understand why, I searched on the net the fact that I have a bad accuracy with the mnist dataset.
I found some people using the Adam Optimizer. You're using the Adadelta Optimizer.

So, I decided to test your mnist example with the Adam Optimizer and I obtained a 0,96 accuracy which is better than 0,10.
I'm wondering if it's because the Adadelta optimizer doesn't work perfectly or there is another explanation?

Thanks in advance for your help.

**Comment by sousoux** [comment]
The MNIST example is a toy example to show the flow. It was actually taken directly from TensorFlow examples. You should contact the TensorFlow developers to understand how to use their different optimizers.
